pep-9999.rst

```diff
@@ -0,0 +1,520 @@
+that would be one way of testing out new release management tooling.
+
+
+Changes to extension module loading in CPython pre-releases
+-----------------------------------------------------------
+
+Extension modules built against CPython pre-releases prior to the ABI freeze
+for that release series may not actually load correctly on the final release.
+
+Historically, adoption of alpha and beta releases has been low enough that this
+hasn't really been a problem in practice. However, this PEP proposes to actively
+encourage widespread operational use of beta releases, and we'd like to make it
+clear that such binaries are only reliably portable to the exact version that
+was used to build them (they may fail to load on even the very next alpha or
+beta release).
+
+To that end, this PEP proposes amending the extension module ``SOABI`` marker
+for CPython pre-releases to use a "cpython_unstable-" prefix, and only switch
+back to the regular "cpython-" prefix once the ABI for that particular version
+has been frozen on entry to the release candidate stage.
+
+(TODO: Link to https://github.com/ncoghlan/cpython/pull/3)
+(TODO: What's the Windows equivalent of this?)
+
+
+Example Future Release Schedules
+================================
+
+Under this proposal, Python 3.9.0b1 would be released in December 2019, two
+months after the Python 3.8.0 baseline feature release in October 2019.
+
+The 3.9.0b2 release would then follow 2 months later in February 2020,
+continuing through to 3.9.0b9 in April 2021.
+
+3.9.0rc1 would be published in June 2021, 3.9.0rc2 in July 2021, and then
+then the full release published as 3.9.0 in August 2021.
+
+The cycle would start over again in October 2021, with the publication
+of 3.10.0b1.
+
+Assuming maintenance releases of 3.9.x were also to occur every other month
+(offset from the 3.10.0 beta releases), the overall release timeline
+would look like:
+
+* 2019-12: 3.9.0b1
+* ... beta releases every other month
+* 2021-04: 3.9.0b9
+* 2021-06: 3.9.0rc1 (feature freeze, ABI freeze, pyc format freeze)
+* 2021-07: 3.9.0rc2
+* 2021-08: 3.9.0
+* 2021-09: 3.9.1, 3.8.x (final 3.8.x binary maintenance release)
+* 2021-10: 3.10.0b1
+* 2021-11: 3.9.2
+* 2021-12: 3.10.0b2
+* ... beta and maintenance releases every other month
+* 2023-04: 3.10.0b10
+* 2023-05: 3.9.11
+* 2023-06: 3.10.0rc1 (feature freeze, ABI freeze, pyc format freeze)
+* 2023-07: 3.10.0rc2, 3.9.12
+* 2023-08: 3.10.0
+* 2023-09: 3.10.1, 3.9.13 (final 3.9.x binary maintenance release)
+* 2023-10: 3.11.0b1
+* ... etc
+
+
+(The exact schedule of maintenance releases would be up to the release team -
+alternating months with the rolling beta releases is just one possibility)
+
+There are always two or three active maintenance branches in this model,
+which preserves the status quo in that respect.
+
+
+Caveats and Limitations
+=======================
+
+Actual release dates may be adjusted up to a month earlier or later at
```

Comment:
Let's be real: we never release early 😄 

But we should be clearer about two things here:
* which releases? Beta? Or maintenance releases? Or both?
* "dates may be adjusted" - perhaps "scheduled"? As in, announcing ahead of time that the release won't be at the usual week